docs/Getting Started/Integration Basics/index.md

```diff
@@ -1236,3 +1236,201 @@ The `SDLSessionBean` should be inside a Java package other than the default pack
+
+```js
+export default {
+    "entrypoint": "./index.html",
+    "appIcon": "./app_icon.png",
+    "appId": "hello-webengine",
+    "appName": "Hello WebEngine",
+    "category": "DEFAULT",
+    "additionalCategories": [],
+    "locales": {
+        "de_DE": {
+            "appName": "Hallo JS",
+            "appIcon": "./app_icon.png"
+        }
+    },
+    "appVersion": "1.0.0",
+    "minRpcVersion": "6.0",
+    "minProtocolVersion": "5.0"
+};
+```
+
+The entrypoint's value should have the same name as your app's HTML file. The `LifecycleConfig` file has a `loadManifest` method that takes in a manifest file and sets up the configuration itself. With these changes, the `index.html` may look something like this:
+
+```html
+<html>
+    <head>
+        <script src='./SDL.min.js'></script>
+    </head>
+    <body>
+        <script type='module'>
+            import sdl_manifest from './manifest.js';
+
+            const lifecycleConfig = new SDL.manager.LifecycleConfig()
+                .loadManifest(sdl_manifest)
+                .setLanguageDesired(SDL.rpc.enums.Language.EN_US);
+
+            lifecycleConfig.setTransportConfig(new SDL.transport.WebSocketClientConfig());
+
+            const appConfig = new SDL.manager.AppConfig()
+                .setLifecycleConfig(lifecycleConfig);
+
+            const managerListener = new SDL.manager.SdlManagerListener()
+                .setOnStart((sdlManager) => {
+                    // managers are ready
+                })
+                .setOnError((sdlManager, info) => {
+                    console.error('Error from SdlManagerListener: ', info);
+                });
+
+            const sdlManager = new SDL.manager.SdlManager(appConfig, managerListener)
+                .start()
+                .addRpcListener(SDL.rpc.enums.FunctionID.OnHMIStatus, (onHmiStatus) => {
+                    // HMI Level updates
+                    const hmiLevel = onHmiStatus.getHmiLevel();
+                    console.log("Current HMI Level: ", hmiLevel);
+                });
+        </script>
+    </body>
+</html>
+```
+
+Note that the manifest file is using the import/export module syntax, and so in the HTML file it is imported in as a module. Also note that in the transport configuration the parameters for `WebSocketClientConfig` are empty. This is because for WebEngine apps those connection details are expected as query parameters in the URL. See below for an example of what the URL is expected to be once the server is running. `sdl-host` is the location of the SDL Core WebSocket server. `sdl-port` is the port of that WebSocket server. `sdl-transport-role` refers to SDL Core's role, which is as a server (as opposed to a client).
```

Comment:
```suggestion
Note that the manifest file is using the import/export module syntax; in the HTML file it is imported in as a module. Also note that in the transport configuration the parameters for `WebSocketClientConfig` are empty. This is because for WebEngine apps those connection details are expected as query parameters in the URL. See below for an example of what the URL is expected to be once the server is running. `sdl-host` is the location of the SDL Core WebSocket server. `sdl-port` is the port of that WebSocket server. `sdl-transport-role` refers to SDL Core's role, which is as a server (as opposed to a client).
```